### DISCUSSION TITLE
Cursor locked in place when distance snapping hit object

### DISCUSSION THREAD

**Discussion by brainage04** [ppy/osu#28117] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I was trying to figure out how to adjust the distance snap in the editor, when my cursor suddenly stopped moving.

It seems that when you start holding Alt and THEN start holding Left Click to move the hit object, the cursor becomes locked in place and does not move until Alt is released, and another input packet is sent from the mouse.

Holding Left Click and THEN holding Alt will snap the object properly.

I am aware that part of this bug could potentially be a problem with my mouse. For reference, I am using a Logitech G502 wired mouse with default on-board settings (except for DPI changes and keybind changes unrelated to Left Click).

I have attached a video demonstration of this bug (in the form of a YouTube link, because my file is bigger than 10MB) below with audio, so that distinct key presses on my keyboard and mouse can be heard.

### Screenshots or videos

https://youtu.be/X9fV-eBJy2M

### Version

2024.412.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15210118/compressed-logs.zip)

**Comment by bdach** [comment, marked as answer]
Can't reproduce here I'm afraid. You're going to have to debug your setup I think.

**Reply by brainage04** [reply]
I'm sure you can tell from the logs but I was using a Linux-based operating system in this video (Linux Mint). I will try with Windows 10 and Arch Linux and update you if I can reproduce it again or not.
